src/NServiceBus.Core/Pipeline/BehaviorContext.cs

```diff
@@ -19,6 +20,11 @@ public IBuilder Builder
             }
         }
 
+        public Task RaiseNotification<T>(T notification) where T : INotification
+        {
+            return Builder.Build<EventAggregator>().Raise(notification);
```

Comment:
In my personal humble opinion when the aggregator becomes an important part of the pipeline it should be explicitly required by the root context. The owner of the aggregator would be the startable endpoint like with the pipeline cache until we introduce a service locator.